src/js/component/panel-title/Title.js

```diff
@@ -4,13 +4,30 @@ import PropTypes from 'prop-types';
 const propTypes = {
     /** The text to display */
     text: PropTypes.string.isRequired,
+
+    /** Number of panel issues */
+    count: PropTypes.number,
+};
+
+const defaultProps = {
+    count: null,
 };
 
 const Title = props => (
-    <h3 className="panel-title">{props.text}</h3>
+    <div>
+        <h3 className="panel-title">
+            {props.text}
+            {props.count !== null && (
+                <>
+                    {` (${props.count})`}
+                </>
```

Comment:
I tried to make this prettier but since React keeps telling me to put each prop statement in a different line, I am not sure how to improve it. If you have a better suggestion, I would love to hear it!

Reply:
Would this work?

```
{`${props.text} ${props.count !== null ? props.count : ''}`}
```

Reply:
Ohh that actually worked, thank you so much! I didn't think of wrapping the whole thing into backticks